java/main/src/main/java/com/saucelabs/saucebindings/options/SauceLabsOptions.java

```diff
@@ -135,6 +143,15 @@ protected void setCapability(String key, Object value) {
                 prerunMap.put(Prerun.valueOf(keyString), val);
               });
       setPrerun(prerunMap);
+    } else if ("tags".equals(key)) {
```

Comment:
Does this need special handling? If setTag() is adding it to current tags field, then doesn't the default getTags() return what we need?